skyportal/utils/offset.py

```diff
@@ -728,22 +734,34 @@ def get_formatted_standards_list(
             | (df["ra_float"] <= ra_filter_range[1])
         ]
 
+    if standard_type == "ESO":
+        mag = []
+        for _, row in df.iterrows():
+            commentSplit = row["comment"].split(" ")
+            mag.append(float(commentSplit[1].replace("V=", "")))
+        df["mag"] = mag
+
+        df = df[(df["mag"] <= magnitude_range[0]) & (df["mag"] >= magnitude_range[1])]
```

Comment:
Is magnitude range always going to be [faint, bright] or is it possible that the order would be [low, high]?

Reply:
I am adding some checking to be sure they always go faint, bright

Reply:
I would prefer that you just order them as you want, and let the user input them either way.

Reply:
Ok I fixed that.